projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -275,9 +275,11 @@ jQuery( document ).ready( function ( $ ) {
 			};
 
 			if ( window.jpTracksAJAX ) {
-				window.jpTracksAJAX.record_ajax_event(eventName, 'click', eventProp);
+				window.jpTracksAJAX.record_ajax_event(eventName, 'click', eventProp).always( function() {
+					window.location.href = button.attr('href');
+				} );
```

Comment:
The change kinda works, but I think we'll always want to redirect the user (not depending on `window.jpTracksAJAX`), so I'd move this out of this `if`

Reply:
It's also not considering cases where we have `target` set to `_blank`, and we'd eventually lose other attributes as well